lib/orderbook/OrderBook.ts

```diff
@@ -83,14 +83,7 @@ class OrderBook {
       for (const [index, match] of matches.entries()) {
```

Comment:
* You don't use `index` anymore, so you can remove it, together with `entries()`.
* You can aggregate all the `updateOrderQuantity` async calls into one `Promise.all`, because they are not dependent on each other. It might perform faster.